home/screens/QRCodeScreen.tsx

```diff
@@ -68,7 +72,7 @@ export default function BarCodeScreen(props) {
     if (Platform.OS === 'ios') {
       props.navigation.pop();
     } else {
-      props.navigation.goBack(null);
```

Comment:
This null was intentional IIRC. I'd look into this more closely.

Reply:
https://reactnavigation.org/docs/upgrading-from-4.x#goback
We're on 5.x already so this parameter wasn't doing anything.